src/zino/tasks/vendor.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from typing import Optional, Tuple
+
+from zino import state
+from zino.snmp import SNMP
+from zino.tasks.task import Task
+
+_logger = logging.getLogger(__name__)
+ENTERPRISES = (1, 3, 6, 1, 4, 1)
+
+
+class VendorTask(Task):
+    """Fetches and stores state information about a Device's vendor"""
+
+    async def run(self):
+        vendor = await self._get_enterprise_id()
+        _logger.debug("Device enterprise id: %r", vendor)
```

Comment:
Shouldn't we also log the device name here? Otherwise this log entry doesn't help much

Reply:
Indeed. I'm used to the plugin loggers in nav/ipdevpoll, that automatically include the name of the current job and device in the logs :)